src/doc/book/associated-types.md

```diff
@@ -11,7 +11,7 @@ this:
 trait Graph<N, E> {
     fn has_edge(&self, &N, &N) -> bool;
     fn edges(&self, &N) -> Vec<E>;
-    // etc
+    // Etc.
```

Comment:
Well, since et cetera means there are other things, let's write "Etc..." instead of just "Etc." ;)